packages/instant/src/components/standard_sliding_panel.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import { SlideAnimationState, StandardSlidingPanelContent, StandardSlidingPanelSettings } from '../types';
+
+import { InstallWalletPanelContent } from './install_wallet_panel_content';
+import { SlidingPanel } from './sliding_panel';
+
+export interface StandardSlidingPanelProps extends StandardSlidingPanelSettings {
+    onClose: () => void;
+}
+
+export class StandardSlidingPanel extends React.Component<StandardSlidingPanelProps> {
+    public render(): React.ReactNode {
+        const { animationState, content, onClose } = this.props;
+        return (
+            <SlidingPanel animationState={animationState} onClose={onClose}>
+                {this._getNodeForContent(content)}
+            </SlidingPanel>
+        );
+    }
+    private readonly _getNodeForContent = (content: StandardSlidingPanelContent): React.ReactNode => {
+        switch (content) {
+            case StandardSlidingPanelContent.InstallWallet:
+                return <InstallWalletPanelContent />;
+            default:
```

Comment:
I'd like to get rid of this `default:` to ensure that if we add a new `StandardSlidingPanelContent.` there is a proper React.ReactNode returned here. Thoughts?

Reply:
Yea we have these defaults all over the place because of the tslint config. However, I think for instant it makes sense to turn this off since it is useful not to specify a default in certain cases. The only gotcha is that if we have a value coming externally (ie. non typescript-land) we need to consider that and include the default. However, internally it is super useful to switch on typescript enums without providing a default so went ahead and did that.

Reply:
Awesome, thx!